util/dvsim/CfgJson.py

```diff
@@ -167,7 +169,8 @@ def set_target_attribute(path, target, key, dict_val):
         target[key] = dict_val
         return
 
-    # Neither value looks like a default. Raise an error.
-    raise RuntimeError('{!r}: Value for key {!r} is {!r}, but '
-                       'we already had a conflicting value of {!r}.'
-                       .format(path, key, dict_val, old_val))
+    # both fields are not default. Always uses leaf most field and raise warnings rather than RuntimeError
+    target[key] = old_val
+    log.warning('{!r}: Value for key {!r} is {!r}, but '
+                'we already had a conflicting value of {!r}.'
+                .format(path, key, dict_val, old_val))
```

Comment:
@sriyerg for future reference.

Reply:
I don't see this change in the commit. Is it removed now?